specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2022-10-01/examples/ChildAvailabilityStatuses_List.json

```diff
@@ -0,0 +1,149 @@
+{
+  "parameters": {
+    "resourceUri": "resourceUri",
+    "api-version": "2022-10-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/227b734f-e14f-4de6-b7fc-3190c21e69f6/resourceGroups/JUHACKETRHCTEST/providers/Microsoft.Compute/virtualMachineScaleSets/rhctest/virtualMachines/4/providers/Microsoft.ResourceHealth/availabilityStatuses/current",
+            "name": "current",
+            "type": "Microsoft.ResourceHealth/AvailabilityStatuses",
+            "location": "southcentralus",
+            "properties": {
+              "availabilityState": "Available",
+              "summary": "There aren't any known Azure platform problems affecting this virtual machine",
+              "detailedStatus": "",
+              "reasonType": "",
+              "recommendedActions": [
+                {
+                  "action": "If you're having problems, use the Troubleshoot tool to get recommended solutions.",
+                  "actionUrl": "<#TroubleshootV2Blade>",
+                  "actionUrlText": "Troubleshoot tool"
+                },
+                {
+                  "action": "If you are experiencing problems you believe are caused by Azure, contact support",
+                  "actionUrl": "<#SupportCase>",
+                  "actionUrlText": "contact support"
+                }
+              ],
+              "occuredTime": "2018-11-30T23:37:44Z",
+              "reasonChronicity": "Transient",
+              "reportedTime": "2018-12-05T19:45:45.6640838Z"
+            }
+          },
+          {
+            "id": "/subscriptions/227b734f-e14f-4de6-b7fc-3190c21e69f6/resourceGroups/JUHACKETRHCTEST/providers/Microsoft.Compute/virtualMachineScaleSets/rhctest/virtualMachines/4/providers/Microsoft.ResourceHealth/availabilityStatuses/2018-11-30+23%3a36%3a03Z",
+            "name": "2018-11-30+23%3a36%3a03Z",
+            "type": "Microsoft.ResourceHealth/AvailabilityStatuses",
+            "location": "southcentralus",
+            "properties": {
+              "availabilityState": "Unavailable",
+              "summary": "This virtual machine is rebooting as requested by an authorized user or process. It will be back online after the reboot completes.",
+              "detailedStatus": "",
+              "reasonType": "Customer Initiated",
+              "rootCauseAttributionTime": "2018-11-30T23:36:02.514Z",
+              "recommendedActions": [
+                {
+                  "action": "Check back here for status updates",
+                  "actionUrl": "",
+                  "actionUrlText": ""
```

Comment:
Curious question. How accurate is this. Would they actually be returned as empty strings, or would they just be null or omitted from the response?